client/js/dayData/day-data-creator.js

```diff
@@ -0,0 +1,60 @@
+/////////////////////////////////////////////////////////////////
+//   Hopefully we can continually modify this code as we change
+//   the schema, so we can always have sweet data generators!
+///////////////////////////////////////////////////////////////////
```

Comment:
Is the idea to move these data creation files into our backend once we build that out this week?